docs/source/yelpsoa_configs.rst

```diff
@@ -463,9 +463,9 @@ out of the load balancer, so it justifies having less sensitive thresholds.
 ``marathon-[clustername].yaml``
 -------------------------------
 
-e.g. ``marathon-norcal-prod.yaml``, ``marathon-mesosstage.yaml``. The
+e.g. ``marathon-pnw-prod.yaml``, ``marathon-mesosstage.yaml``. The
```

Comment:
Is `mesosstage` still around?

Reply:
nope! i half-debated re-writing this, but i think we're waiting to just delete all the mesos mentions in one swoops so i decided against investing too much time/effort into cleaning mentions of marathon up

(maybe once someone mentions they got confused by what mesosstage/marathon are after reading this :p)